tests/hash-backend-performance/src/session.ts

```diff
@@ -0,0 +1,69 @@
+import { Configuration, FrontendApi } from "@ory/client";
+
+import type { BeforeRequest, RequestParams } from "./types";
+
+const SESSION_ID_REFRESH_INTERVAL = 1000 * 60; // 60 seconds
```

Comment:
Why do we expire the session every 60 seconds? This isn't what happens in practice (sessions are very long-lived, you can get the actual expiry date from `session.expires_at`)

I don't think it would make a huge difference to the scenario but I wouldn't bother to refresh the token unless the session had actually expired

Reply:
I took the `expires_at` from kratos into account. If it's not set I assume there is no expiration date. I think this will become an issue, though, if the expiration date is very close to the current date as further requests will probably not be valid anymore? Anyway, if this becomes an issue (e.g. if we shorten the lifetime) we can still adjust it.
Mainly, this was here to show a usage of the hooks :)

Reply:
Cool, thank you!